projects/openNeuroClient/openNeuroClient.py

```diff
@@ -11,46 +12,76 @@
 # import project modules from rt-cloud
 from rtCommon.utils import loadConfigFile, stringPartialFormat
 from rtCommon.clientInterface import ClientInterface
+from rtCommon.bidsArchive import BidsArchive
 
 # path for default configuration toml file
-defaultConfig = os.path.join(currPath, 'conf/openNeuro.toml')
+defaultConfig = os.path.join(currPath, 'conf/openNeuroClient.toml')
 
 
 def doRuns(cfg, bidsInterface, subjInterface, webInterface):
-    # TODO - fill in processing of bids dataset
-    pass
+    """
+    Process a run of a bids dataset. The subject and run configuration
+    values will be supplied by the cfg parameter.
+    Args:
+        cfg: configurations parsed from the project toml config file
+        bidsInterface: client interface to bids data
+        webInterface: client interface to user web page
+    Returns: no return value
+    """
+    subject = cfg.subjectName
+    run = cfg.runNum[0]
+    entities = {'subject': subject, 'run': run, 'suffix': 'bold', 'datatype': 'func'}
+    webInterface.clearRunPlot(run)
+    if cfg.writeBidsArchive is True:
+        # Create a new bids archive from the incrementals
+        bidsArchivePath = os.path.join('/tmp', 'bids_archive_' + uuid.uuid4().hex)
```

Comment:
Might be better to use `tempfile` interface here for cross-platform compatibility.